pkgs/development/python-modules/cysignals/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, cython
+, sphinx
+}:
+
+buildPythonPackage rec {
+  pname = "cysignals";
+  version = "1.6.9";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "003invnixqy1h4lb358vwrxykxzp15csaddkgq3pqqmswnva5908";
+  };
+
+  hardeningDisable = [
+    "fortify"
```

Comment:
Can you add a comment about what error is prevented with that?

Reply:
There are explicit checks for that, no specific failure:

```
gcc -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -DCYTHON_CLINE_IN_TRACEBACK=0 -U_FORTIFY_SOURCE -Isrc/cysignals -Ibuild/src -Ibuild/src/cysignals -I/nix/store/l9j2jsc9flrbmpf799nw9wdq59gpwms8-python-2.7.14/include/python2.7 -c build/src/cysignals/signals.c -o build/temp.linux-x86_64-2.7/build/src/cysignals/signals.o
In file included from build/src/cysignals/signals.c:602:0:
build/src/cysignals/implementation.c:27:2: error: #error "cysignals must be compiled without _FORTIFY_SOURCE"
 #error "cysignals must be compiled without _FORTIFY_SOURCE"
  ^~~~~
In file included from build/src/cysignals/signals.c:602:0:
build/src/cysignals/implementation.c:66:1: warning: function declaration isn't a prototype [-Wstrict-prototypes]
 static cysigs_t cysigs;
 ^~~~~~
```